### PR TITLE
Dropship sidedoors

### PR DESCRIPTION
**Pull request by Doubleumc**
# About the pull request

Replaces the pairs of 1-wide sidedoors on dropships with a 2-wide sidedoor. Icons are identical to the 1-wide sidedoors with the door part widened.

# Explain why it's good for the game

Two less moving parts that might accidentally get opened just as a lockdown or launch is initiated thus locking it open at the worst possible time. Also, I think it looks cleaner.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Before
image
image

After
image
image

</details>

# Changelog
:cl:
imageadd: 2-wide dropship side doors
maptweak: changed dropship side doors to a lone doublewide door
/:cl:

### CODE REVIEW

File: maps/shuttles/dropship_alamo.dmm
Comment (Nanu308) [inline maps/shuttles/dropship_alamo.dmm:1]: lgtm

File: maps/shuttles/dropship_normandy.dmm
Comment (Nanu308) [inline maps/shuttles/dropship_normandy.dmm:1]: lgtm

**Review by Nanu308 — COMMENTED** [review]
mapping lgtm

**Review by fira — COMMENTED** [review]
Looks great. Might want to also edit the crashed ship on CC for consistency

**Review by harryob — APPROVED** [review]

**Comment by Doubleumc** [comment]
Swapped the doors on the CC crashed ship (Typhoon) as well, good catch. Adjusted their dirs to align the hitbox with the sprite; multitile doors are weird.

**Pull request merged**